### DISCUSSION TITLE
你们会不会500啊？

### DISCUSSION THREAD

**Discussion by superen7527** [chatgpt-web-dev/chatgpt-web#283] [Q&A]
我真的服了，用原版只有前端那个，之前改了docker容器内部一个图标标题 就报错500需要重装，换了现在这个项目的可以改标题我就没动过docker内部的东西，有一天升级pull完了以后又500楼，我还以为是更新了配置文件，对着文件一行一行重新改改了一个多小时一直重新试还是不行一直500，崩溃。请问有人和我一样吗？要怎么解决啊，或者需要我怎么提供日志？

**Comment by superen7527** [comment]
logs看了一下 只有这个500

ChatGPTError: ChatGPT error 500:
    at fetchSSE (file:///app/node_modules/.pnpm/chatgpt@5.1.2/node_modules/chatgpt/build/index.js:55:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  statusCode: 500,
  statusText: '',
  [cause]: Response {
    size: 0,
    [Symbol(Body internals)]: {
      body: [PassThrough],
      stream: [PassThrough],
      boundary: null,
      disturbed: true,
      error: null
    },
    [Symbol(Response internals)]: {
      type: 'default',
      url: 'http://172.17.0.1:3001/conversation',
      status: 500,
      statusText: '',
      headers: [Object],
      counter: 0,
      highWaterMark: 16384
    }

**Comment by Kerwin1202** [comment]
没看出，你是不是设置了反代啊，http://172.17.0.1:3001/conversation' 这个 然后反代有问题？

**Reply by superen7527** [reply]
还真是，不过我也想过这个问题，可是你搭建命令改不了反代，而且我模式用的api，然后这个设置会存在数据库里面缓存外面改不了了，那个反代路径改了我现在进不去界面改不了，同时改不了就进不去界面，死循环了。

关键是之前原项目，反代还是正常的时候，也会这样。不知道为什么。然后现在，我换了个数据库文件夹重新搭建，还是500，不明白出问题在哪里。

如果是反代路径问题应该怎么解决？如果是别的问题，我应该提供哪些东西？

**Reply by Kerwin1202** [reply]
搭建命令改不了反代？ 其实可以，但是仅限首次搭建 而且你这个 172.xx 明细是你自己改过的吧 默认不是这个

这些设置管理员界面可以修改的，至于你说的改不了不太明白


后面的 不太明白你的意思，需要就提供怎么搭建的以及搭建文件 然后解图看看

至于修改这里修改就可以了
<img width="699" alt="image" src="https://github.com/Kerwin1202/chatgpt-web/assets/16319833/781bfe77-47b0-4e5a-9896-8cb61b8b419e">

**Reply by superen7527** [reply]
我表达的可能不清楚，我的意思是已经进不去管理界面了。所以只能通过搭建命令改，但是我发现之前在管理界面改过的地方好像会记录下来怎么改搭建命令都没用。你看方不方便留个什么邮箱还是什么，我把网站发给你看看？

不过我感觉应该不是反代问题，因为我项目用的api没有用反代，而且反代五百也不至于全部界面五百。

**Reply by Kerwin1202** [reply]
如果是显示 500 报错，那么 人工是docker 部署 需要提供 docker log 不然我也看不出，邮箱 zhiweidada@qq.com
